exercises/practice/hamming/.meta/lib/example.dart

```diff
@@ -1,9 +1,5 @@
 class Hamming {
   int distance(String a, String b) {
-    if ((a.isEmpty && b.isNotEmpty) || (a.isNotEmpty && b.isEmpty)) {
-      throw ArgumentError('no strand must be empty');
-    }
-
     if (a.length != b.length) {
       throw ArgumentError('left and right strands must be of equal length');
```

Comment:
The language in testing seems to be changed from left/right to first/second? If that's intentional, we should be consistent here as well.